### DISCUSSION TITLE
How to use this plugin.

### DISCUSSION THREAD

**Discussion by AlphabetsAlphabets** [st3v3nmw/obsidian-spaced-repetition#429] [Q&A]
As the title says, I have no idea how to even start using the plugin. I've checked the "notes" section in the wiki, but it doesn't actually do anything except add front matter at the top. I found a new button on the side bar on the left of obsidian, but when I open it it shows that it is empty.

**Comment by AlphabetsAlphabets** [comment]
I've managed to get a few flashcards setup, but now I'm trying to create different decks but it isn't clear how to do that at all.

**Reply by dylandoherty** [reply]
In the Spaced Repetition Plugin Settings, you can add multiple tags under "Flashcard tags". For each tag, you get a different deck.

**Reply by st3v3nmw** [reply]
Hi @AlphabetsAlphabets, did you check this project's wiki? It has some details on that
